lib/sles4sap.pm

```diff
@@ -517,14 +517,30 @@ and threads that it can create.
 sub test_pids_max {
     # UserTasksMax should be set to "infinity" in /etc/systemd/logind.conf.d/sap.conf
     my $uid = script_output "id -u $sapadmin";
+
     # The systemd-run command generates syslog output that may end up in the console, so save the output to a file
```

Comment:
evaluate to update this comment to explain why the execution is within a `sh` script

Reply:
**systemd-run** command string is too long, it is not stable to type the command in terminal and failed to execute the command, it is more stable to execute the command in script

Reply:
I have updated the comment